src/test/java/uk/gov/digital/ho/hocs/workflow/processes/FOI_ALLOCATION.java

```diff
@@ -77,25 +77,24 @@ public void happyPath() {
         }).execute();
 
         //then
-        verify(processScenario, times(1)).hasCompleted(CHOOSE_REQUEST_TYPE);
         verify(processScenario, times(1)).hasCompleted(CHOOSE_FOI_HUB);
         verify(bpmnService).updateAllocationNoteWithDetails(eq(CASE_UUID), eq(STAGE_UUID), eq(ALLOCATION_MESSAGE),
                 eq("ALLOCATE"), eq(ACCEPTANCE_TEAM_UUID), eq(STAGE_UUID));
+        verify(processScenario, times(1)).hasCompleted(CONFIRMATION_SCREEN);
     }
 
     @Test
     public void happyPathWithBack() {
```

Comment:
If you take out the back flow, you will need to remove this test as well. There may be others you'll need to check.

Reply:
Done